### PR TITLE
chore: update ajax e2e tests - network capture

### PR DESCRIPTION
**Pull request by patrickhousley**
Please add a one-paragraph summary here, suitable for a release notes description. This will help with documentation.
---

### Overview

- Updated all the `specs/ajax/*` tests to use network capturing. This resolves race conditioning issues within our tests.
- Updated cors settings on testing server to prevent caching and support logging of `OPTIONS` calls.
- Update server logging to log debug shim messages without having to specify the `-L` flag.
- Testing server network capture
  - Removing the `clear` method because of it's flakiness due to race conditions.
  - Fixed an issue where tests where not properly failing when the capture `await` was not resolving.
- WDIO
  - Removed the `quiet` flag due to no usage.
  - Improved testing server logging so the testing server can produce logs separate from WDIO verbose mode. The `-L` flag no longer implies `-v`.
  - Added a new test logger that will provide more verbose indication of which browser is being ran per thread and will log out the test name when each test in a spec file starts.
  - Added better error handling to the testing server and network capture connectors.
  - Re-enabling `network` and `terminal` log capturing when `-D` flag is provided.
  - Added code to clear the LT tunnel log before starting tests and enable verbose tunnel logging.

### Related Issue(s)

https://new-relic.atlassian.net/browse/NR-287577

### CODE REVIEW

**Review by metal-messiah — DISMISSED** [review]

File: tools/wdio/runner.mjs [hunk 1]
```diff
@@ -28,11 +29,25 @@ const configFilePath = path.join(
...
+  '.lambdatest'
+)
+fs.ensureDirSync(lambdaTestOutputDir)
+const lambdaTestTunnelLog = path.join(
+  lambdaTestOutputDir,
+  'lt.log'
+)
+if (fs.pathExistsSync(lambdaTestTunnelLog)) {
+  fs.removeSync(lambdaTestTunnelLog)
+}
```
Comment (cwli24) [inline tools/wdio/runner.mjs:50]: Shouldn't do this. Leave it manual. It may also cause issues when executing 2 runners simul

Comment (patrickhousley) [reply tools/wdio/runner.mjs:50]: You mean executing wdio with the `-T` flag in two different terminals?

Comment (patrickhousley) [reply tools/wdio/runner.mjs:50]: Honestly, I added this to make capturing tunnel logs easier so I could give the right log to LT team. IDC either way but I would just point out that the LT plugin also downloads the tunnel binary to the same directory so I wouldn't advise running wdio in two different tunnels with the `-T` flag.

**Review by cwli24 — APPROVED** [review]
👍

**Pull request merged**